lib/pages/chat/chat.dart

```diff
@@ -1545,6 +1544,11 @@ class ChatController extends State<Chat>
     }
   }
 
+  void actionWithClearSelections(Function action) {
+    action();
+    clearSelectedEvents();
+  }
```

Comment:
Hard to understand for me, Why put func inside func?

Reply:
understand for Khai, I also don't know better way
it will reuse `clearSelectedEvents`, make sure it called every action performed

Reply:
I don't want to put clearSelectedEvents after every action I find it redundant so I searched a way to refactor that. For me it's not hard to understand but if you have any ideas tell me